wagtail/models/sites.py

```diff
@@ -1,13 +1,14 @@
 from collections import namedtuple
+from copy import deepcopy
+from typing import List, Tuple, Union
 
 from django.apps import apps
 from django.conf import settings
-from django.core.cache import cache
 from django.core.exceptions import ValidationError
 from django.db import models
-from django.db.models import Case, IntegerField, Q, When
 from django.db.models.functions import Lower
 from django.http.request import split_domain_port
+from django.utils.local import ClearableLocal
```

Comment:
```suggestion
from wagtail.utils.local import ClearableLocal
```

Reply:
Thanks @zerolab! That'll teach me to code while poorly!

Reply:
I hear you! Did that so many times, and every time is a bit like "ah, I am not that poorly, I can do this" 🤦‍♂️